boot.php

```diff
@@ -1133,7 +1133,6 @@ function is_backend() {
 		$backend[] = "noscrape";
 		$backend[] = "p";
 		$backend[] = "photo";
```

Comment:
In order to support private photos I believe 'photo' has to support sessions as well.